README.md

```diff
@@ -1,91 +1,134 @@
 # Bazel Rules for Fuzz Tests
 
-This repository contains [Bazel](https://bazel.build/)
-[Starlark extensions](https://docs.bazel.build/versions/master/skylark/concepts.html)
-for defining fuzz tests in Bazel projects. 
+## Overview
 
-**This is not an officially supported Google product.**
+This repository contains [Bazel](https://bazel.build/) [Starlark extensions](https://docs.bazel.build/versions/master/skylark/concepts.html) for defining fuzz tests in Bazel projects.
```

Comment:
nit: I feel overall that this top-level README is a bit too wordy. Perhaps would be better to move parts of this to links to more detailed places? 

Ideally the README should just be a few lines showing a very simple example on how to get started very quickly. 

For more detailed information, might be better to move these other sections to sub-pages, especially sections like "Defining fuzzing engines" which should be less common but take up a lot of space here.

Reply:
I am open to structuring this documentation differently. The reason I went with this approach is that this seems to be the prevailing form of documentation for Bazel rules. See for example:

* https://github.com/bazelbuild/rules_python
* https://github.com/bazelbuild/rules_go
* https://github.com/bazelbuild/rules_kotlin

And there are others like that, too.

I don't feel too strongly about this, so let me know if you prefer to take a different approach here.